src/cmd/serve.rs

```diff
@@ -268,6 +269,12 @@ fn create_new_site(
     site.enable_serve_mode();
     site.set_base_url(base_url);
     if let Some(output_dir) = output_dir {
+        if !force && output_dir.exists() {
```

Comment:
I would swap the 2 conditions just because it would be clearer imo

Reply:
This pattern honestly was taken straight from [the build command's](https://github.com/getzola/zola/blob/695c17d2222e2ee3f83d304d0828265b1c7407cf/src/cmd/build.rs#L17) pattern of ifs. 

Do you still think it's worth changing?

Reply:
Nah it's fine